docs/custom_quantum_functions.md

```diff
@@ -148,40 +157,29 @@ bool led_update_kb(led_t led_state) {
         } else {
             writePinHigh(B4);
         }
-        return true;
     }
+    return res;
 }
 ```
 
 ### Example `led_update_user()` Implementation
 
+This incomplete example, would play a sound if capslock was turned on or off. It returns true, because you also want the LEDs to maintain their state.
```

Comment:
```suggestion
This incomplete example would play a sound if Caps Lock is turned on or off. It returns `true`, because you also want the LEDs to maintain their state.
```